### PR TITLE
Delete/update comment functionality

### PR DESCRIPTION
**Pull request by scottwainstock**
This adds the ability to delete/update machine comments that you made.

### CODE REVIEW

File: app/models/machine_condition.rb [hunk 1]
```diff
@@ -13,6 +13,22 @@ class MachineCondition < ApplicationRecord
 
   scope :limited, (-> { order('created_at DESC').limit(MachineCondition::MAX_HISTORY_SIZE_TO_DISPLAY) })
 
+  def update(options = {})
+    if (options[:comment])
+      self.comment = options[:comment]
+
+      if (
```
[the remark this answers is not in the document]
Comment (fiveNinePlusR) [reply app/models/machine_condition.rb]: it's actually the code below where I tagged it that I am talking about... sorry

[the remark this answers is not in the document]
Comment (RyanTG) [reply app/models/machine_condition.rb]: Just a note that there is no "above the fold" first comment anymore. All comments are tucked under the "Show Comments" button. The first one isn't visible by default.

[the remark this answers is not in the document]
Comment (fiveNinePlusR) [reply app/models/machine_condition.rb]: is there anything like that on the mobile app? if not i guess you could just delete this code and that field from the db.

**Comment by scottwainstock** [comment]
@fiveNinePlusR Thanks! Addressed here: https://github.com/pinballmap/pbm/pull/1578

**Pull request merged**